updating/index.adoc

```diff
@@ -88,6 +88,11 @@ xref:../updating/updating-hardware-on-nodes-running-on-vsphere.adoc#updating-har
 * xref:../updating/updating-hardware-on-nodes-running-on-vsphere.adoc#updating-virtual-hardware-on-vsphere_updating-hardware-on-nodes-running-in-vsphere[Updating virtual hardware on vSphere]
 * xref:../updating/updating-hardware-on-nodes-running-on-vsphere.adoc#scheduling-virtual-hardware-update-on-vsphere_updating-hardware-on-nodes-running-in-vsphere[Scheduling an update for virtual hardware on vSphere]
 
+[id="updating-clusters-overview-sro"]
+== Updating a cluster that includes the SRO
+
+xref:../updating/updating-clusters-sro.adoc#updating-clusters-sro[Updating a cluster that includes the SRO]: When updating a cluster that includes the SRO, it is important to consider whether the new kernel module version will be compatible with the kernel modules currently loaded by the SRO. You can run a preflight check to confirm that the SRO will be able to upgrade the kernel modules.
```

Comment:
You could change "that the SRO will be able to upgrade" to use the present tense:

> You can run a preflight check to confirm if the SRO can upgrade the kernel module.

The example also uses "if" instead of "that", because at this stage the result is uncertain.

Also, I have used the singular "kernel module". Please ignore that if more than one is being upgraded.

Reply:
OK